### ISSUE TITLE
Get embeddings from text

### ISSUE THREAD

**Issue by sonic182** [elixir-nx/bumblebee#206]
Is there a way of obtaining embeddings from text? for example, to extract the 768 dim's from a given text using bert model

**Comment by trodrigu** [comment]
https://github.com/elixir-nx/bumblebee/issues/100 may have some things that can help

**Comment by trodrigu** [comment]
https://github.com/trodrigu/text_embeddings_elixir/ as an example

**Comment by rakshans1** [comment]
```python
from transformers import CLIPModel, CLIPProcessor
import torch

model = CLIPModel.from_pretrained("openai/clip-vit-base-patch32")
processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32")

text = "Hello, world!"
inputs = processor(text=text, return_tensors="pt")
model.get_text_features(**inputs)
```

Is it possible to build servings for  `get_text_feature` / `get_image_features`?

**Comment by jonatanklosko** [comment]
@rakshans1 you can do both text and image the same way as the snippet above (featurizer/tokenizer + running base model). And yeah, we will have serving for both text and image.
